Marlin/configuration_store.cpp

```diff
@@ -787,8 +787,6 @@ void Config_ResetDefault() {
   #if ENABLED(PIDTEMP)
     #if ENABLED(PID_PARAMS_PER_HOTEND) && HOTENDS > 1
       HOTEND_LOOP()
```

Comment:
I decided not to change this particular instance, because the PID macros already ignore the second parameter when `HOTENDS == 1`.